portfolio/.theia/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-    "java.configuration.updateBuildConfiguration": "automatic"
+    "java.configuration.updateBuildConfiguration": "automatic",
```

Comment:
In the future, make sure that you don't commit configuration changes like these. (It will be more important when you work on the same repository with other people.)